terraform/modules/happy-stack-eks/happy_config.tf

```diff
@@ -0,0 +1,9 @@
+data "happy_resolved_app_configs" "configs" {
+  app_name    = local.secret["tags"]["project"]
```

Comment:
is there a reason you use `local.secret["tags"]["project"]` instead of `var.app_name`?

Reply:
I don't think var.app_name exists. We could explicitly require the user provide the variable in their stack, but I think it might be easier to infer it from the environment. Maybe this is a bad idea?

Reply:
oh right, it exists for happy-stack-ecs but not happy-stack-eks, seems like if one requires it they both should 🤷

Reply:
or both shouldn't. I'm tempted to take the interface from EKS and adopt it to ECS since EKS has users already and no one is on the ECS one yet

Reply:
@alexlokshin-czi will probably also want to pass it in though because he's thinking about multi-app clusters.